enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -319,12 +319,18 @@ func (s *ExecutionServer) Dispatch(ctx context.Context, req *repb.ExecuteRequest
 		}
 	}
 
+	taskGroupID := "ANON"
```

Comment:
can we const-ify this since we use it in the disk cache too?

Reply:
Done.